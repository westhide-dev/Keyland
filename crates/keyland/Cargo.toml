[package]
description       = "keyland"
edition.workspace = true
license.workspace = true
name              = "keyland"
version.workspace = true


[dependencies]
