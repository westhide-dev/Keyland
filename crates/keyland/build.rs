use kcommon::nil::{Nil, NIL};
use kerror::KResult;

mod scripts;

fn main() -> KResult<Nil> {
    println!("Keyland build.rs Init");

    let profile = std::env::var("PROFILE")?;
    println!("Profile type: {profile}");

    println!("Keyland build.rs Done");
    Ok(NIL)
}
