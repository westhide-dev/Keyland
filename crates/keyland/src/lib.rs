pub mod config;
pub mod nil;

pub mod prelude;
