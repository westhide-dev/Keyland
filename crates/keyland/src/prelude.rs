pub use crate::nil::{Nil, NIL};
