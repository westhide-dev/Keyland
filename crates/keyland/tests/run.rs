use insta::assert_snapshot;
use keyland::prelude::*;

#[test]
fn nil() {
    assert_snapshot!(format!("{NIL:?}"));
}
