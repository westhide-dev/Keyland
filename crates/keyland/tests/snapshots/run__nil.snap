---
source: crates/keyland/tests/run.rs
expression: "format!(\"{NIL:?}\")"
---
()
